Home.md

```diff
@@ -44,11 +55,11 @@ Just go to [http://demo.dswarm.org](http://demo.dswarm.org) and try it out or se
 ### Installation and Configuration
 
 It might be a good idea to run d:swarm locally to get full insight into the processes of our application.
-Installation instructions can be found in the [[Server Install]] or [[Developer Install]] guide, [[d:swarm Configuration|dswarm Configuration]] provides details on how to configure the system.
+Installation instructions can be found in the [[Developer Install]] guide, [[d:swarm Configuration|dswarm Configuration]] provides details on how to configure the system. (For productive use of d:swarm see [[Server Install]].)
 
 ### Running the System
 
-Once installed, the local front end (usually) runs at `http://localhost:9999` and the local data hub browser may be accessed at `http://localhost:7474/browser/`. Hint: see [[Cypher Cheat Sheet]] for details on Neo4j. You may also want to have a look at the [[MySQL Cheat Sheet]] for our metadata repository schema (see also our [[domain model|Domain Model]]) and use a tool of your choice to explore the database.
+Once installed, the BackOffice (usually) runs at `http://localhost:9999`.  You may want to have a look at the [[MySQL Cheat Sheet]] for our metadata repository schema (see also our [[domain model|Domain Model]]) and use a tool of your choice to explore the database.
 
 ## Contributing

```

Comment:
maybe we should move the links to mailing and issue tracker upfront (i.e. before we link to the project structure and coding guidelines)